singleapplication.cpp

```diff
@@ -235,9 +235,10 @@ QString SingleApplication::currentUser() const
  * Sends message to the Primary Instance.
  * @param message The message to send.
  * @param timeout the maximum timeout in milliseconds for blocking functions.
+ * @param block block until the primary application exits
```

Comment:
Probably forgot to update the description?
```suggestion
 * @param sendMode mode of operation
```